layouts/fastly/helix.vcl

```diff
@@ -1000,6 +1006,13 @@ sub vcl_deliver {
 sub vcl_error {
   set req.http.X-Trace = req.http.X-Trace + "; vcl_error";
 #FASTLY error
+
+  if (obj.status == 301) {
+    set obj.http.Content-Type = "text/html";
+    set obj.http.Location = req.http.X-Location;
+    synthetic "Moved Permanently";
```

Comment:
Nothing _really_ wrong with this because most clients will automatically redirect, but spec does say:
> Unless the request method was HEAD, the entity of the response SHOULD contain a short hypertext note with a hyperlink to the new URI

I'd be tempted to save some bytes and give an empty response, or do as per spec, or feel free to ignore this comment.

Reply:
```suggestion
    synthetic "Moved permanently <a href='" + req.http.X-Location+ "'>" + req.http.X-Location + "</a>";
```

Would this satisfy the spec?